crates/ruff_linter/src/rules/pydocstyle/rules/triple_quotes.rs

```diff
@@ -60,18 +73,28 @@ pub(crate) fn triple_quotes(checker: &mut Checker, docstring: &Docstring) {
     match expected_quote {
         Quote::Single => {
             if !leading_quote.ends_with("'''") {
-                checker.diagnostics.push(Diagnostic::new(
-                    TripleSingleQuotes { expected_quote },
+                let mut diagnostic =
+                    Diagnostic::new(TripleSingleQuotes { expected_quote }, docstring.range());
+
+                diagnostic.set_fix(Fix::safe_edit(Edit::range_replacement(
+                    prefixes + &format!("'''{}'''", docstring.body().as_str()),
```

Comment:
Is this always safe? Could this ever change the inner contents of the string?

Reply:
ran it on the cpython source with `--fix`, looks fine, but I trust your judgement!

Reply:
I stand corrected, it breaks when confronted with this condition:

```
'''Ensure that the given "actual" string ends with "exp_end"'''
```